### PR TITLE
#21 Move endpoints to config file and add Base Url for remote services User and Gamification

### PR DESCRIPTION
**Pull request by artem-zhukov**
connect to #21

### CODE REVIEW

File: src/main/java/juja/microservices/gamification/slackbot/dao/RestGamificationRepository.java [hunk 1]
```diff
@@ -42,7 +50,7 @@ public String saveDailyAchievement(DailyAchievement daily) {
         HttpEntity<DailyAchievement> request = new HttpEntity<>(daily, setupBaseHttpHeaders());
         String result = "";
         try {
-            ResponseEntity<String> response = restTemplate.exchange(URL_SEND_DAILY, HttpMethod.POST, request, String.class);
+            ResponseEntity<String> response = restTemplate.exchange(BASE_URL+URL_SEND_DAILY, HttpMethod.POST, request, String.class);
```
Comment (NikolayNN) [inline src/main/java/juja/microservices/gamification/slackbot/dao/RestGamificationRepository.java]: to do code format according to code convention. add spaces.

File: src/main/java/juja/microservices/gamification/slackbot/dao/RestUserRepository.java [hunk 2]
```diff
@@ -2,24 +2,28 @@
...
 
-
+@PropertySource("application.properties")
 public class RestUserRepository implements UserRepository {
 
     private final RestTemplate restTemplate;
-    private final String URL_GET_USER = "/users/search";
+
+    @Value("${user.baseURL}")
+    private String BASE_URL;
```
Comment (NikolayNN) [inline src/main/java/juja/microservices/gamification/slackbot/dao/RestUserRepository.java]: according to code convention BASE_URL means constant. if you want use BASE_URL you have to use "private final String"

Comment (artem-zhukov) [reply src/main/java/juja/microservices/gamification/slackbot/dao/RestUserRepository.java]: Agree. I didn't think about it.

**Review by adaleorion — APPROVED** [review]
well done.

**Pull request merged**